.github/ISSUE_TEMPLATE.md

```diff
@@ -1,35 +1,22 @@
-**Note: for support questions, please use one of these channels:**
-https://github.com/angular/material2/blob/master/CONTRIBUTING.md#question.
-This repository's issues are reserved for feature requests and bug reports.
+#### Bug, feature request, or proposal:
 
-* **Do you want to request a *feature* or report a *bug*?**
 
+#### What is the expected behavior?
 
 
-* **What is the current behavior?**
+#### What is the current behavior?
 
 
+#### What are the steps to reproduce?
 
-* **If the current behavior is a bug,
-please provide steps to reproduce and if possible a minimal demo of the problem**
-via https://plnkr.co or similar.
+Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
+Plunker template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd
 
-Plunker Template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd?p=preview
 
+#### What is the use-case or motivation for changing an existing behavior?
 
-* **What is the expected behavior?**
 
+#### What are the relevant versions of Angular, Material, OS, affected browsers, etc.?
```

Comment:
`#### What browser are you using? What's your Material and Angular versions?`

Just that "relevant versions" is up for interpretation and people will just say "Firefox" or omit versions they don't think is relevant.